client/src/components/Workflow/Editor/Node.vue

```diff
@@ -49,11 +49,7 @@
                     triggers="hover"
                     placement="bottom"
                     :show.sync="popoverShow">
-                    <Recommendations
-                        v-if="popoverShow"
-                        :step-id="id"
-                        :datatypes-mapper="datatypesMapper"
-                        @onCreate="onCreate" />
+                    <Recommendations :step-id="id" :datatypes-mapper="datatypesMapper" @onCreate="onCreate" />
```

Comment:
These were super slow, emitting a ton of events also when the option isn't enabled, is there a way this can work with v-if instead of show ?

Reply:
A thin wrapper around the recommendation component may work here, I will give that a try.

Reply:
seems to work